datafusion/functions/src/string/contains.rs

```diff
@@ -108,34 +108,19 @@ pub fn contains(args: &[ArrayRef]) -> Result<ArrayRef, DataFusionError> {
         (Utf8View, Utf8View) => {
             let mod_str = args[0].as_string_view();
             let match_str = args[1].as_string_view();
-            let res = regexp_is_match::<
-                StringViewArray,
-                StringViewArray,
-                GenericStringArray<i32>,
-            >(mod_str, match_str, None)?;
-
+            let res = arrow_contains(mod_str, match_str)?;
```

Comment:
Could you update the doc for `contains`?

Reply:
yeah, I update doc of `contains`
https://github.com/zhuliquan/datafusion/blob/5d0d86a0e0ede41530d2b2b587cdb6f96b7da499/datafusion/functions/src/string/contains.rs#L105